src/vm/linux/types.rs

```diff
@@ -96,3 +96,59 @@ impl Default for Capabilities {
         }
     }
 }
+
+/// TDX specific VM initialization information
+#[derive(Debug)]
+#[repr(C)]
+pub struct InitVm {
+    /// Guest TD attributes
+    pub attributes: u64,
+
+    /// Software-defined ID for non-owner-defined configuration of the guest TD
+    /// (runtime or OS configuration)
+    pub mrconfigid: [u64; 6],
+
+    /// Software-defined ID for the guest TD’s owner
+    pub mrowner: [u64; 6],
+
+    /// Software-defined ID for owner-defined configuration of the guest TD
+    /// (specific to the workload)
+    pub mrownerconfig: [u64; 6],
+
+    /// reserved for future extensibility
+    reserved: [u64; 1004],
+
+    /// direct configuration of CPUID leaves/subleaves virtualization
+    pub cpuid_nent: u32,
+
+    cpuid_padding: u32,
```

Comment:
```suggestion
    _padding: u32,
```

The  `_` prefix indicates that this field is not intended to be used, and is strictly for padding bytes.

Reply:
fixed